### DISCUSSION TITLE
Xournal+Wacom+Linux: Settings discussion (what is input stabilization etc.)

### DISCUSSION THREAD

**Discussion by bugsbunny88** [xournalpp/xournalpp#3721] [Q&A]
Can folks share with us their settings that work for them? I'm having issues and its because a lot of the times I don't fully understand the settings ; for instance the Input stabilization ; if i choose an averaging method: gaussian ; ... is that how folks are using it ? and what does prepocessor do ? how about pressure sensitivity etc ; what do you guys use on defaul t?

**Comment by Max0uishere** [comment]
I use these settings with Wacom CTH-460:

![image](https://user-images.githubusercontent.com/102471538/161067853-f25da640-3846-454d-bca9-9c5cc5c03e71.png)
![image](https://user-images.githubusercontent.com/102471538/161067943-23e04f46-b08b-4bf1-9fa0-5fdc42893426.png)

If you only write sentences, you may want to use "Velocity based Gaussian weights" for "Averaging method" where σ = 1.2. It makes handwriting cursive. I recommend the settings in the pictures for mixed usage.

**Reply by mhalano** [reply]
@Max0uishere What configuration software are you using in this screenshot?

**Reply by Technius** [reply]
These are from the Preferences window of Xournal++. 
* "Input Devices", "Options", "Input Stabilization" can be found in the "Input System" tab
* "Pressure Sensitivity" can be found in the "Stylus" tab
